joomla/joomla-cms
# [4.2] Convert the checkfiles plugin to service provider

### Summary of Changes
Converts the checkfiles plugin to service provider and adds some unit tests. This pr is based on #38060, so please test that one first.

### Testing Instructions
- Create a folder in images with the name test and upload there a big image from unsplash or so which is bigger than 1080 pixels
- Go to the task scheduler back end list
- Click on new "Image Size Check"
- Select the test image in the task parameters
- Save the task and close the form
- Run the task by clicking on "Run Test" button

### Actual result BEFORE applying this Pull Request
The image in the test folder has a width or height of 1080 pixels.

### Expected result AFTER applying this Pull Request
The image in the test folder has a width or height of 1080 pixels.
